api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -243,52 +241,46 @@ public String createAllOfUsBillingProject(String billingProjectName, String serv
   @Override
   public void deleteBillingProject(String billingProjectName) {
     BillingV2Api billingV2Api = serviceAccountBillingV2ApiProvider.get();
-    retryHandler.run(
+    rawlsRetryHandler.run(
         (context) -> {
-          billingV2Api.deleteBillingProject(billingProjectName);
+          billingV2Api.deleteBillingProjectV2(billingProjectName);
           return null;
         });
   }
```

Comment:
I think methods like this that only use Rawls endpoints should be moved into a Rawls service, since they are no longer relying on firecloud-orchestration.

Reply:
ditto to other fireclou vs rawls comment. Happy to chat offline about this topic.